test/files/pos/t10387.scala

```diff
@@ -1,269 +0,0 @@
-object foo {
```

Comment:
Thanks. I had been debating removing this from about a day after I introduced it, but I didn't want to make a "delete a test" PR.